### PR TITLE
[sciter-js] Updated sciter-js to 4.4.8.31

### PR DESCRIPTION
**Pull request by VuYeK**
**Describe the pull request**
 
 * #### What does your PR fix?
 
 Bumps the version of sciter-js from 4.4.8.16 to 4.4.8.31. In new version there is no sciter-sqlite.dll.
 
 * #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?
 
 all, No
 
 * #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?
 
 Yes.
 
 * #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?
 
 Yes.
 
 **If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by VuYeK** [comment]
Something is wront with this tests

**Comment by FrankXie05** [comment]
@VuYeK  Looks like there is a problem with the CI test, I will re-request the test and identify the problem. :)

**Comment by JackBoosY** [comment]
Close and re-open this PR to re-active our PR suggestion agent.

**Comment by JackBoosY** [comment]
@VuYeK Can you please open a new PR to commit this changes? It seems our agent has some problems.
Sorry for that.

**Comment by VuYeK** [comment]
@JackBoosY I have closed and reopened PR, still some errors

**Comment by JackBoosY** [comment]
@VuYeK So can you please open another PR to commit this changes?

File: ports/sciter-js/vcpkg.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "sciter-js",
-  "version-string": "4.4.8.16",
+  "version-string": "4.4.8.31",
```
Comment (JackBoosY) [inline ports/sciter-js/vcpkg.json:3]:
```suggestion
  "version": "4.4.8.31",
```

File: versions/s-/sciter-js.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "c1ab21374bbcd674911ff5093e8ce859334f7352",
+      "version-string": "4.4.8.31",
```
Comment (JackBoosY) [inline versions/s-/sciter-js.json:5]:
```suggestion
      "version": "4.4.8.31",

```

File: versions/s-/sciter-js.json [hunk 3]
```diff
@@ -1,5 +1,11 @@
 {
   "versions": [
+    {
+      "git-tree": "c1ab21374bbcd674911ff5093e8ce859334f7352",
```
Comment (JackBoosY) [inline versions/s-/sciter-js.json:4]:
```suggestion
      "git-tree": "708c5345421be4b92ea288a7447a4c228fc98413",

```

**Review by FrankXie05 — APPROVED** [review]

File: versions/s-/sciter-js.json [hunk 4]
```diff
@@ -1,5 +1,12 @@
 {
   "versions": [
+    {
+      "git-tree": "708c5345421be4b92ea288a7447a4c228fc98413",
+
+      "version": "4.4.8.31",
+
+      "port-version": 0
```
Comment (JackBoosY) [inline versions/s-/sciter-js.json:8]:
```suggestion
      "git-tree": "708c5345421be4b92ea288a7447a4c228fc98413",
      "version": "4.4.8.31",
      "port-version": 0
```

**Review by JackBoosY — APPROVED** [review]

**Review by JackBoosY — APPROVED** [review]

**Pull request merged**